### PR TITLE
Mati test

### PR DESCRIPTION
**Pull request by mat-ilde**
Adding .gitignore file

### CODE REVIEW

**Comment by mat-ilde** [comment]
can u please review?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+# Eclipse artifacts, including WTP generated manifests
+bin
+.classpath
+.project
+.settings
+.springBeans
+spring-*/src/main/java/META-INF/MANIFEST.MF
+
+# IDEA artifacts and output dirs
```
Comment (gsantam) [inline .gitignore:43]: Since we are not going to work with IDEA, we better remove these ignores

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+*.iml
+*.ipr
+*.iws
+.idea
+out
+test-output
+atlassian-ide-plugin.xml
+
+# VS Code
```
Comment (gsantam) [inline .gitignore:52]: Same as above

**Review by gsantam — APPROVED** [review]
LGTM

**Pull request merged**